daphne_worker/src/durable/report_store.rs

```diff
@@ -270,15 +270,15 @@ impl DurableObject for ReportStore {
 }
 
 pub(crate) fn nonce_hex_from_report(report_hex: &str) -> Option<&str> {
```

Comment:
Reviewer note: This function borrows the hex encoding of the nonce from a report. The definition of the nonce has changed to not encompass the timestamp, hence this change. Note that this change does not reflect renaming the nonce to "report ID" and moving it in the report structure. This change will come later in the PR stack.